### ISSUE TITLE
BCH Wallet private keys

### ISSUE THREAD

**Issue by pekatete** [blockchain/blockchain-wallet-v4-frontend#1687]
***Do***
I tried to get the private keys for my wallets using the recovery phrase. I am able to get the private keys to my Bitcoin (BTC) wallet address  but I am unable to get the Bitcoin Cash (BCH) wallet address private keys.

I am using this tool: https://github.com/iancoleman/bip39

**NOTE:** I am able to get the BTC wallet address private key, I need to get the BCH one(s). I have tried different indexes and accounts (in the derivation path) to no avail. 
**PS:** I was advised by customer support (on the twitter) to open an issue here.

**Comment by plondon** [comment]
> my Bitcoin (BTC) wallet address but I am unable to get the Bitcoin Cash (BCH) wallet address private keys

They are the same thing

If you are using iancoleman he is using 145 for the purpose. So yes they will be different on there. But your BTC private keys are your BCH private keys.

**Comment by pekatete** [comment]
No the addresses are not the same. If I convert my BCH wallet q address to the native BTC one, they are not the same. And nope, he is using 144 (in any case you can change it!

**Comment by plondon** [comment]
> If I convert my BCH wallet q address to the native BTC one, they are not the same

When you say your BCH wallet q address are you just opening request and comparing it to the BTC request address? Because yes they may be different addresses because the receive index could be different.

> And nope, he is using 144

<img width="1353" alt="Screen Shot 2019-06-21 at 11 40 01 AM" src="https://user-images.githubusercontent.com/5640772/59934359-56087980-9419-11e9-8470-3dc45c685b5b.png">


> PS: I did explain this on the twitter

I'm trying to help. I don't have all of the information you do. I don't know all of the things you have tried. I do not have your address list.

**Comment by pekatete** [comment]
> When you say your BCH wallet q address are you just opening request and comparing it to the BTC request address? Because yes they may be different addresses because the receive index could be different.

This is exactly what I explained to you yesterday, I checked at least 50 rows on different account (and index) settings but the tool could not generate neither the (converted) BCH address nor the private key.

> I'm trying to help. I don't have all of the information you do. I don't know all of the things you have tried. I do not have your address list.

I think we've got to the point where my question just needs to be answered plainly. **How do I get the private keys to my Blockchain.info BCH wallet address?** If your assertion is I am using the wrong tool, then what tool should I use (and are there sufficient instructions using the said tool)?

**Comment by plondon** [comment]
1. create wallet
2. import mnemonic to iancoleman tool https://iancoleman.io/bip39/
3. scroll down to addresses
4. see address, public, private keys

**your BTC private keys are your BCH private keys.**

**Comment by pekatete** [comment]
I already explained that the BTC addresses DO NOT convert to the BCH q address, therefore, the private keys for those BTC addresses are not the ones for the BCH wallet addresses. I have actually tried it using the very same tool you link to (but you suggested is the wrong one earlier).
I looked at 50+ BTC addresses and none of them convert to the BCH address in the wallet. Even converting the BCH address(es) in my blockchain.info wallet back to native bitcoin addresses, none matches the ones listed by the tool.
So my question stands.
